### ISSUE TITLE
[FEATURE] Discover pokestops on GUI selected location

### ISSUE THREAD

**Issue by Dragunovam** [jabbink/PokemonGoBot#731]
**Description:**
Whenever I select a location for the bot to run to, it only catches pokemon along the way and once it gets to the location, it runs back to the random pokestop selection near the starting pokestop location.

I suggest a way to start a new pokestop discovery in the direct proximity where you directed the bot to, so it can harvest in the new location for pokestops/pokemon.

**Comment by pashorizer** [comment]
It already does that. Set `allow_leave_start_area=true`

**Comment by Dragunovam** [comment]
You're right, it does allow for some kind of discovery of new pokestops, but they tend to be close to the cluster of already visited pokestops. If I make my bot run 1 km away from the starting cluster, it will usually run back 1 km to near the cluster it last visited
